database/PracticeOperation/0007_AddColumn_Patient_AvatarLink.sql

```diff
@@ -0,0 +1,13 @@
+USE [PatientCheckIn]
```

Comment:
Ading new script is a good approach for running database. Since we have not deploy the code to any prod database, we should update the existing sql file creating Patient table instead of adding new script altering it.

Reply:
Updated CreateTable script and removed new script